.actions/assistant.py

```diff
@@ -18,10 +18,11 @@
 import shutil
 import tempfile
 import urllib.request
+from collections.abc import Iterable, Iterator, Sequence
 from itertools import chain
 from os.path import dirname, isfile
 from pathlib import Path
-from typing import Any, Dict, Iterable, Iterator, List, Optional, Sequence, Tuple
+from typing import Any, Optional
```

Comment:
Given the amount of changes and expecting a fair number of merge conflicts, I'd recommend merging all other patches immediately ready first and then push this.

Reply:
@t-vi is it a good time to land it now?